vizro-core/src/vizro/models/_dashboard.py

```diff
@@ -114,14 +118,41 @@ def build(self):
             fluid=True,
         )
 
+    def _make_page_layout(self, page):
```

Comment:
```suggestion
    def _make_page_layout(self, page: Page):
```
would probably make things a bit easier to understand and type check.